invertedindex/core_index.go

```diff
@@ -0,0 +1,74 @@
+package invertedindex
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/filecoin-project/go-indexer-core"
+
+	"github.com/multiformats/go-multihash"
+
+	"github.com/filecoin-project/dagstore/shard"
+)
+
+var ErrNotFound = errors.New("multihash not found in Index")
+
+var _ Index = (*IndexerCoreIndex)(nil)
+
+type IndexerCoreIndex struct {
```

Comment:
I think this type can be private. Try renaming `NewIndexerCore` to `index.NewInverted()`.